test/Parse/ConditionalCompilation/avrTargetWithStdlib.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %swift -typecheck %s -verify -target avr-none-none -disable-objc-interop -wmo -enable-experimental-feature Embedded
+
+import Swift
+
+#if arch(avr) && os(none) && _runtime(_Native) && _endian(little)
```

Comment:
did you mean to include "&& _pointerBitWidth(_16)" here? (Im not suggesting you should or should not)

Reply:
yeah, makes sense, I'll add that with the next fix/push, cheers :)